app/controllers/account/pages_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Account::PagesController < ApplicationController
+  def index
+    @pages = collection
+  end
+
+  def show
+    @page = resource
+  end
+
+  private
+
+  def resource
+    Page.find(params[:id])
+  end
+
+  def collection
+    Page.page(params[:page]).per(10)
```

Comment:
collection returns all
and pagination is implemented in INDEX

@pages = collection.page(params[:page]).per(10)